src/message/messageActionSheet.js

```diff
@@ -54,77 +56,95 @@ const copyToClipboard = async ({ _, auth, message }) => {
   showToast(_('Message copied'));
 };
 copyToClipboard.title = 'Copy to clipboard';
+copyToClipboard.errorMessage = 'Failed to copy message to clipboard';
 
 const editMessage = async ({ message, dispatch }) => {
   dispatch(startEditMessage(message.id, message.subject));
 };
 editMessage.title = 'Edit message';
+editMessage.errorMessage = 'Failed to edit message';
 
 const deleteMessage = async ({ auth, message, dispatch }) => {
   if (isAnOutboxMessage(message)) {
     dispatch(deleteOutboxMessage(message.timestamp));
   } else {
-    api.deleteMessage(auth, message.id);
+    await api.deleteMessage(auth, message.id);
   }
 };
 deleteMessage.title = 'Delete message';
+deleteMessage.errorMessage = 'Failed to delete message';
 
-const unmuteTopic = ({ auth, message }) => {
-  api.unmuteTopic(auth, message.display_recipient, message.subject);
+const unmuteTopic = async ({ auth, message }) => {
+  await api.unmuteTopic(auth, message.display_recipient, message.subject);
 };
 unmuteTopic.title = 'Unmute topic';
+unmuteTopic.errorMessage = 'Failed to unmute topic';
 
-const muteTopic = ({ auth, message }) => {
-  api.muteTopic(auth, message.display_recipient, message.subject);
+const muteTopic = async ({ auth, message }) => {
+  await api.muteTopic(auth, message.display_recipient, message.subject);
 };
 muteTopic.title = 'Mute topic';
+muteTopic.errorMessage = 'Failed to mute topic';
 
-const unmuteStream = ({ auth, message, subscriptions }) => {
+const unmuteStream = async ({ auth, message, subscriptions }) => {
   const sub = subscriptions.find(x => x.name === message.display_recipient);
   if (sub) {
-    api.toggleMuteStream(auth, sub.stream_id, false);
+    await api.toggleMuteStream(auth, sub.stream_id, false);
   }
 };
 unmuteStream.title = 'Unmute stream';
+unmuteStream.errorMessage = 'Failed to unmute stream';
 
-const muteStream = ({ auth, message, subscriptions }) => {
+const muteStream = async ({ auth, message, subscriptions }) => {
   const sub = subscriptions.find(x => x.name === message.display_recipient);
   if (sub) {
-    api.toggleMuteStream(auth, sub.stream_id, true);
+    await api.toggleMuteStream(auth, sub.stream_id, true);
   }
 };
 muteStream.title = 'Mute stream';
+muteStream.errorMessage = 'Failed to mute stream';
 
-const starMessage = ({ auth, message }) => {
-  api.toggleMessageStarred(auth, [message.id], true);
+const starMessage = async ({ auth, message }) => {
+  await api.toggleMessageStarred(auth, [message.id], true);
```

Comment:
It's not obvious, but this is `await`ing on void.

(That's really a bug in `toggleMessageStarred`, though.)